### PR TITLE
[MARTIFACT-32] Add new plugins issues/fixes

### PR DESCRIPTION
**Pull request by michael-o**
https://issues.apache.org/jira/browse/MARTIFACT-32

### CODE REVIEW

File: src/main/resources/org/apache/maven/plugins/artifact/buildinfo/not-reproducible-plugins.properties [hunk 1]
```diff
@@ -19,59 +19,38 @@
 
 # default org.apache.maven.plugins groupId
 maven-archetype-plugin=3.2.0
-# https://issues.apache.org/jira/browse/ARCHETYPE-590
```
Comment (hboutemy) [inline src/main/resources/org/apache/maven/plugins/artifact/buildinfo/not-reproducible-plugins.properties:22]: why do you remove this explanation of what was the issue that has been fixed?
I think it's useful for people wanting to understand

File: src/main/resources/org/apache/maven/plugins/artifact/buildinfo/not-reproducible-plugins.properties [hunk 2]
```diff
@@ -19,59 +19,38 @@
...
-org.apache.felix+maven-bundle-plugin=5.1.4
-# https://issues.apache.org/jira/browse/FELIX-6495
-# https://issues.apache.org/jira/browse/FELIX-6496
-org.apache.karaf.tooling+karaf-maven-plugin=fail:https://issues.apache.org/jira/browse/KARAF-7367
+org.apache.felix+maven-bundle-plugin=5.1.5
+org.apache.karaf.tooling+karaf-maven-plugin=4.3.7
 org.apache.nifi+nifi-nar-maven-plugin=1.3.4
-# https://issues.apache.org/jira/browse/NIFI-9857
-org.apache.servicemix.tooling+depends-maven-plugin=fail:https://issues.apache.org/jira/browse/SM-5021
+org.apache.servicemix.tooling+depends-maven-plugin=1.5.0
```
Comment (hboutemy) [inline src/main/resources/org/apache/maven/plugins/artifact/buildinfo/not-reproducible-plugins.properties:41]: issue has been fixed in SCM, but release not done: it's better to show this as unsolved yet

**Review by hboutemy — CHANGES_REQUESTED** [review]

File: src/main/resources/org/apache/maven/plugins/artifact/buildinfo/not-reproducible-plugins.properties (hunk 1 above)
Comment (michael-o) [reply src/main/resources/org/apache/maven/plugins/artifact/buildinfo/not-reproducible-plugins.properties:22]: Issue is addressed and in all information can be retrieved from the SCM log, no?

File: src/main/resources/org/apache/maven/plugins/artifact/buildinfo/not-reproducible-plugins.properties (hunk 2 above)
Comment (michael-o) [reply src/main/resources/org/apache/maven/plugins/artifact/buildinfo/not-reproducible-plugins.properties:41]: You are right, I will revert.

**Comment by michael-o** [comment]
@hboutemy Please have a look again. Do you still want me to retain comments although they are available in SCM?

File: src/main/resources/org/apache/maven/plugins/artifact/buildinfo/not-reproducible-plugins.properties (hunk 1 above)
Comment (hboutemy) [reply src/main/resources/org/apache/maven/plugins/artifact/buildinfo/not-reproducible-plugins.properties:22]: it's really hard to find such info

Comment (michael-o) [reply src/main/resources/org/apache/maven/plugins/artifact/buildinfo/not-reproducible-plugins.properties:22]: OK, I will drop the hunk and retain the information.

**Comment by michael-o** [comment]
@hboutemy Comments restored.

File: src/main/resources/org/apache/maven/plugins/artifact/buildinfo/not-reproducible-plugins.properties [hunk 3]
```diff
@@ -48,10 +47,11 @@ maven-acr-plugin=3.2.0
 
 org.antlr+antlr3-maven-plugin=fail:https://github.com/antlr/antlr3/pull/195
 
-org.apache.felix+maven-bundle-plugin=5.1.4
+org.apache.felix+maven-bundle-plugin=5.1.5
```
Comment (hboutemy) [inline src/main/resources/org/apache/maven/plugins/artifact/buildinfo/not-reproducible-plugins.properties:50]: why is 5.1.5 required? there should be a link to an issue

**Review by hboutemy — CHANGES_REQUESTED** [review]
thanks for restoring issues comments

File: src/main/resources/org/apache/maven/plugins/artifact/buildinfo/not-reproducible-plugins.properties (hunk 3 above)
Comment (hboutemy) [reply src/main/resources/org/apache/maven/plugins/artifact/buildinfo/not-reproducible-plugins.properties:50]: ok, https://issues.apache.org/jira/browse/FELIX-6496 was already in the file...

**Comment by michael-o** [comment]
So this is fine now?

**Pull request merged**